openreview/venue/venue.py

```diff
@@ -1169,8 +1169,9 @@ def ithenticate_request_similarity_report(self):
         )
 
         for edge in tqdm(edges):
-            edge["values"][0].label = "Similarity Requested"
-            updated_edge = self.client.post_edge(edge["values"][0])
+            e = self.client.get_edge(edge["values"][0]["id"])
```

Comment:
instead of getting the edge again you can do `e = openreview.api.Edge.from_json(edge["values"][0])` to construct the edge object.

Reply:
I've made the change, thanks! 😄